source/index.html

```diff
@@ -2,11 +2,132 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>HTML Academy: Седона</title>
+    <link href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&display=swap" rel="stylesheet">
   </head>
   <body>
+  <!-- Если у нас есть блок в блоке, его название не обязательно начинать с названия блока-родителя?
+   Или не может быть в блоке других блоков, потому что они становятся его элементами?-->
+    <header class="page-header">
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <ul class="main-nav__list">
+          <li class="main-nav__item main-nav__item_active">
+            <a class="main-nav__link">Главная</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="photo.html" class="main-nav__link">Фото и видео</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="form.html" class="main-nav__link">Форма отзыва</a>
+          </li>
+          <li class="main-nav__item">
+            <a href="" class="main-nav__link">HTML Academy</a>
+          </li>
+        </ul>
+        <button type="button" class="main-nav__button-close">Закрыть меню</button>
+      </nav>
+      <a class="header__logo">
+        <img src="https://via.placeholder.com/150x150" width="20" height="20" alt="Логотип сайта">
+      </a>
+    </header>
+    <main class="page-main">
+      <h1 class="visually-hidden">Добро пожаловать в Седону</h1>
+      <!-- Вставляю svg с разными шрифтами, а при стилизации с пом background-image
+       дорисовываю фотографию с горным ландшафтом? Логотип - это все надписи или только sedona?
+       На других стр сохраняется только это одно слово. Или это вообще не логотип, а просто
+       другая часть шапки, ведь в шапке сверху уже есть треугольный логотип?
+       И офтоп, но этот вопрос еще всплывет, забывала задать: чем неразрывный пробел лучше
+       переноса строки <br> (и лучше ли)? -->
+      <img src="https://via.placeholder.com/150x150" width="282" height="214" alt="Добро пожаловать в Седону">
+      <section class="advantages">
+        <h2 class="advantages__header">Седона &ndash; небольшой городок в Аризоне, заслуживающий большего!</h2>
+        <p class="advantages__text">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+        <!-- Разметила так, но можно же и просто как отдельные article или списком их сделать?
+             Список внутрь dl нельзя, только что увидела, исправлю -->
+        <dl class="advantages__list">
+          <div class="advantage__item">
```

Comment:
```suggestion
          <div class="advantages__item">
```

Опечатка. Или ошибка, но с непривычки.
У нас нет блока `advantage` же, но зато есть `advantages` - так что каждый `item` это элемнет `advantages`